### ISSUE TITLE
Diff update colour is not accessible

### ISSUE THREAD

**Issue by brynwhyman** [silverstripe/silverstripe-versioned-admin#96]
Possibly an opinionated issue, but it's difficult to see 'updated' diff colour in the History Viewer. That's the green highlighting used to reflect an update.

According to the [webaim.org color contrast checker](https://webaim.org/), "WCAG 2 level AA requires a contrast ratio of at least ... 3:1 for graphics and user interface components (such as form input borders)." I've taken this to refer to text highlighting too.

Using the CMS colours in the webaim.org color contrast checker tool, it fails with a ratio of 1.07:1. [Example here](https://webaim.org/resources/contrastchecker/?fcolor=F0FFF0&bcolor=F1F3F6)
I've used background colour of `F1F3F6` and a foreground (highlight colour) of `F0FFF0`

**Comment by ScopeyNZ** [comment]
Something prompted both you and @clarkepaul 😆 : #97 .

I've copied your comment and will close this in favour of the other (literally only because the other one was already labelled).
